trojsten/rules/default.py

```diff
@@ -45,6 +47,20 @@ def get_actual_result_rounds(self, competition):
         )
         return rounds.order_by("-end_time", "-number")[:1]
 
+    def grade_text_submit(self, task, user, submitted_text):
+        Grading = namedtuple("Grading", ["response", "points"])
+        solution = [solution.lower() for solution in task.text_submit_solution]
+        if submitted_text in solution:
+            response = "OK"
+            if timezone.now() < task.round.end_time:
```

Comment:
toto by som v default case neriesil...., ostatne sutaze si toto riesia v pravidlach generovania vysledkovky (a obzvlast KSP vysledkovka ocakava, ze spravne submity maju plny pocet bodov, bez ohladu na to, kedy boli zgenerovane a potom podla `end_time` a `second_end_time` su z toho submitu do vysledkovky zaratane bud cele body, cast, alebo nic)